dummy-contract/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
```

Comment:
There are doc comments describing the contracts. I added ReadMe files to all contracts as well now for better documentation.

The purpose of the above contract in particular is:
```
The `dummy-contract` is only used in the integration tests. 

In some tests (e.g. the `registry` integration tests), we need a contract that can be invoked with the `getName` entry point. We cannot use the other available smart contracts because we cause a circular dependency in the `Cargo.toml` file of the `registry` contract if added.
```